### PR TITLE
chore:  print error stack by environment

### PR DESCRIPTION
**Pull request by lianxmfor**
This PR wraps the functions of the pkg/errors package that we use and uses environment variables `DEBUG` in the wrapped methods to determine whether the error should be printed on the call stack

### CODE REVIEW

File: pkg/errdefs/wrap.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+var debug bool
+
+func init() {
+	if os.Getenv("DEBUG") == "enable" {
```
Comment (YikSanChan) [inline pkg/errdefs/wrap.go:15]: 1/0 should be enough.

`DEBUG=1 ...`

Comment (jinghancc) [inline pkg/errdefs/wrap.go:15]: maybe `enabled` would be better?

**Review by jinghancc — APPROVED** [review]
LGTM

**Review by YikSanChan — APPROVED** [review]
lgtm

**Review by wfxr — APPROVED** [review]
LGTM

File: pkg/errdefs/wrap.go (hunk 1 above)
Comment (lianxmfor) [reply pkg/errdefs/wrap.go]: fixed

Comment (lianxmfor) [reply pkg/errdefs/wrap.go]: updated. `DEBUG` equal `true`, `1`, `enable` will print err stack.

**Pull request merged**